source/DrawList.cpp

```diff
@@ -162,7 +162,7 @@ void DrawList::Push(const Body &body, Point pos, Point blur, double cloak, int s
 	item.blur[0] = unit.Cross(blur) / (width * 4.);
 	item.blur[1] = -unit.Dot(blur) / (height * 4.);
 
-	item.alpha = 1. - cloak;
+	item.alpha = (1. - cloak) * body.Alpha(center);
```

Comment:
shouldnt it instead be put into the cloak argument of the function?

Reply:
why?

Reply:
because this feels like a hack lmao

Reply:
nah it isnt, its not really cloak even

Reply:
other functions of body are also used around here

Reply:
also cloak is not dependent on the position of the draw center

Reply:
maybe if we renamed the variable though?